src/babashka/neil.clj

```diff
@@ -500,22 +500,58 @@ will return libraries with 'test framework' in their description.
 See http://github.com/clojars/clojars-web/wiki/Search-Query-Syntax for
 details on the search syntax.")))
 
+(defn dep-search-maven [search-term]
+  (let [url (format
+             "https://search.maven.org/solrsearch/select?q=%s&rows=20&wt=json"
```

Comment:
This number may seem speculative, no? I copypasted it as is from the documentation. Should we leave it at that or maybe increase it, I don't know to 50 or even more? But is it even sensible to try to get more results than this?

Reply:
Don't know to be honest